packages/convenience/contracts/Convenience.sol

```diff
@@ -227,6 +227,7 @@ contract Convenience is Ownable  {
             else
             {
                 delegateState[i] = api3Voting.getVoterState(voteIds[i], delegateAt[i]);
+                voterState[i] = delegateState[i];
```

Comment:
Shouldn't the voterState match the delegateState or am I mistaken?

Reply:
Oops, that line shouldn't be there at all because if the user has a delegate for that vote their voterState is guaranteed to be 0 anyway

Reply:
So one way would have been to keep a single `voterState` and keep the actual voter state if the user didn't have a delegate and keep the delegate state if the user had a delegate. I went with the explicit route instead, keep seperate voter and delegate states and the user of this function will need to decide on which to read based on if the user had a delegate. So according to that convention, the added line is redundant.

Reply:
makes sense, I was receiving an array of 0s from the voterState when a delegate was present but according to the convention above, when a delegate is present we will be ignoring this array. Kind of mixed on returning an array of 0s as well because the 0s signify a state as well.